_fuel_dev_environment/docker/msg-executor/bridge-message-executor/src/main.rs

```diff
@@ -0,0 +1,204 @@
+use chrono::Utc;
```

Comment:
There's `.bin` files. I think they should be deleted and regenerated by the user as needed, no?

Reply:
This will get improved as we flesh out the executor service more